components/server/src/internal/routes/measurement.py

```diff
@@ -15,7 +15,8 @@ def post_measurement(database: Database) -> None:
     """Put the measurement in the database."""
     measurement_data = dict(bottle.request.json)
     metric_uuid = measurement_data["metric_uuid"]
-    if (metric := latest_metric(database, metric_uuid)) is None:
+    report_uuid = measurement_data.get("report_uuid", "")
```

Comment:
Shouldn't the metric be guaranteed to have a report_uuid, making the get unnecessary?

Reply:
As in the collector, yes it should. But requiring less mandatory properties makes the code easier to test. I do feel that this is an interesting discussion though, might be worth having a talk about.

Reply:
See my comments above. Writing that convinced me that we shouldn't use a get() here ;-) 

I'm curious to hear what you think.